isubit/suitcase_interim
# LUGG-828 Universal Grid Styling

See also: https://github.com/isubit/suitcase/pull/52
## Changes
- Added simple jQuery that matches grid cell heights
- Removed grid styling from luggage_people and luggage_resources and moved it to suitcase.css
## Testing

These steps assume that you will be using a local copy of the entomology website and you are using the lugg679 branch of suitcase_config.
- Pull down these changes
- Clear caches (I also run the drush command registry-rebuild, but that might be overkill)
- Go to edit the video view
- Change the video view to an unformatted list with fields
- Edit the unformatted list settings and add luggage-gridrow as a class to the page
- Add luggage-grid as a class under Advanced View settings > Other > CSS Class.
- Remove the weird Title & Body field that’s automatically added to the view and make sure that title and body are being displayed. Rearrange the field order to video, title, then body.
- Save the view
- Look at the video view to see that it is a grid of videos
- Resize the window and make sure that the cells resize accordingly. They should fit the whole window when it is smaller than ~739 pixels wide, text should not be cut off, each row should resize according to the highest cell in that row, no funky text overflow.
